### PR TITLE
Updating favicon and changing toaster/notification behavior

### PR DESCRIPTION
**Pull request by npfitz**
- Updated favicon to 32 x 32. Should address blurriness on higher resolution displays
- Changed behavior to only show the "turn on notifications" toaster if the permission has not yet been set.
- Web notifications will now always show if `Notification.permission === 'granted'`  and the tab is in the background
- needed to add an extra check to the conditions to differentiate between "component has just rendered but hasn't started loading yet", and "query / dashboard has completed loading"

### CODE REVIEW

File: frontend/src/metabase/query_builder/reducers.js [hunk 1]
```diff
@@ -196,12 +197,20 @@ export const uiControls = handleActions(
...
     [CANCEL_QUERY]: {
       next: (state, { payload }) => ({ ...state, isRunning: false }),
     },
     [QUERY_COMPLETED]: {
-      next: (state, { payload }) => ({ ...state, isRunning: false }),
+      next: (state, { payload }) => ({
+        ...state,
+        isRunning: false,
+        isQueryComplete: true,
+      }),
```
Comment (kulyk) [inline frontend/src/metabase/query_builder/reducers.js:213]: Why do we need another flag to track the same state? 🤔

File: frontend/src/metabase/dashboard/containers/DashboardApp.jsx [hunk 2]
```diff
@@ -121,23 +124,23 @@ const DashboardApp = props => {
     if (!isRunning) {
       setIsShowingToaster(false);
     }
-    if (!isRunning && shouldSendNotification) {
+    if (
+      !isRunning &&
+      isLoadingComplete &&
+      Notification.permission === "granted"
+    ) {
```
Comment (kulyk) [inline frontend/src/metabase/dashboard/containers/DashboardApp.jsx:131]: Ain't `isRunning` and `isLoadingComplete` connected here? I wonder if we can check for just one of them?

File: frontend/src/metabase/query_builder/reducers.js (hunk 1 above)
Comment (kulyk) [reply frontend/src/metabase/query_builder/reducers.js:213]: If it's for readability, should we maybe just add a selector like:

```js
const getIsQueryComplete = state => !state.qb.isRunning
```

Comment (kulyk) [reply frontend/src/metabase/query_builder/reducers.js:213]: Oh, got it! Should we maybe replace them with an enum status field? Something like:

```js
type QueryStatus = "idle" | "running" | "complete"

// selectors

const getIsRunning = state => state.qb.queryStatus === "running"
const getIsQueryComplete = state => state.qb.queryStatus === "complete"
```

When we have a few flags in state/reducer reflecting different states of the same process, it's easier to accidentally break them. You can update one flag, but forget to update another one, so the state can become inconsistent. Using a single enum can make it more reliable. WDYT?

Comment (npfitz) [reply frontend/src/metabase/query_builder/reducers.js:213]: I like it! Should also simplify the useEffect hook too. I'll see what I can do

File: frontend/src/metabase/dashboard/containers/DashboardApp.jsx (hunk 2 above)
Comment (npfitz) [reply frontend/src/metabase/dashboard/containers/DashboardApp.jsx]: Simplified with new commit

**Review by ranquild — APPROVED** [review]

**Pull request merged**